src/Util/JsonLD.php

```diff
@@ -227,6 +227,11 @@ private static function fixInvalidJsonLD(array $json): stdClass
 				Logger::debug('schema.org path fixed');
 				$value = 'http://schema.org#';
 			}
+			// Issue 14630: Wordpress Event Bridge uses a URL that cannot be retrieved
+			if ($value == 'https://schema.org/') {
```

Comment:
Please change this line to `if (is_int($key) && $value == 'https://schema.org/') {`

Reply:
Makes sense, done.